src/lgdo/lh5/store.py

```diff
@@ -57,12 +71,21 @@ def gimme_file(self, lh5_file: str | h5py.File, mode: str = "r") -> h5py.File:
             LH5 file name.
         mode
             mode in which to open file. See :class:`h5py.File` documentation.
+        page_buffer
+            enable paged aggregation with a buffer of this size in bytes
+            Only used when creating a new file. Useful when writing a file
+            with a large number of small datasets. This is a short-hand for
+            (fs_stragety="page", fs_pagesize=[page_buffer], fs_persist=True,
+            fs_threshold=1)
+        file_kwargs
+            Keyword arguments for `h5py.File <https://docs.h5py.org/en/stable/high/file.html#reference>`_
```

Comment:
This should work (i.e. create an hyperlink automatically)
```suggestion
            Keyword arguments for :class:`h5py.File`
```